website/src/css/custom.css

```diff
@@ -668,6 +679,14 @@ i.theme-doc-sidebar-item-category.theme-doc-sidebar-item-category-level-2.menu__
   color: var(--ifm-color-gray-900);
 }
 
+.alert--secondary,
+.alert--secondary a,
+.alert--secondary svg {
+  --ifm-alert-background-color: #474748;
+  color: white !important;
+  fill: white !important;
+}
```

Comment:
Do you know of a page where this is used for me to checkout?

Reply:
Here's one with the alert--secondary which the the note alert:
https://docs-getdbt-com-git-fix-insufficient-contrast-o-3250d0-dbt-labs.vercel.app/docs/cloud/connect-data-platform/connect-apache-spark

Reply:
Good catch @JKarlavige. I've added style updates for the blog hrefs.